awx/main/analytics/collectors.py

```diff
@@ -287,8 +292,9 @@ def _copy_table(table, query, path):
                                  main_unifiedjob.elapsed, 
                                  main_unifiedjob.job_explanation, 
                                  main_unifiedjob.instance_group_id
-                                 FROM main_unifiedjob
+                                 FROM main_unifiedjob, main_job
```

Comment:
I think this isn't the type of join we want, this is a type of implicit inner join called an equi join.

Reply:
wouldn't that just require changing the left join below, not this line?

Reply:
:facepalm: nvm